src/main/java/org/opentripplanner/api/model/ApiLeg.java

```diff
@@ -247,9 +247,9 @@
      */
     public Boolean isTransitLeg() {
         if (mode == null) return null;
-        else if (mode.equals(TraverseMode.WALK.toString())) return false;
-        else if (mode.equals(TraverseMode.CAR.toString())) return false;
-        else if (mode.equals(TraverseMode.BICYCLE.toString())) return false;
+        else if (mode.equals("Walk")) return false;
```

Comment:
These should be uppercase, or checked against the enum values themselves

Reply:
This should be fixed when merging with #3031, as it separates out the TraverseMode enum to a new REST API version.

Reply:
This has been fixed now.